src/internal/scheduler/animationFrame.ts

```diff
@@ -50,18 +56,25 @@ export const animationFrame: SchedulerLike = {
          }, delay)
```

Comment:
@benlesh I'm very pro-this-change :-). This is how it should have always worked, definitely an oversight on my part initially.